pkg/api/kubegraph/analysis/rc_test.go

```diff
@@ -0,0 +1,21 @@
+package analysis
+
+import (
+	"testing"
+
+	osgraphtest "github.com/openshift/origin/pkg/api/graph/test"
+	kubeedges "github.com/openshift/origin/pkg/api/kubegraph"
+)
+
+func TestDuelingRC(t *testing.T) {
+	g, _, err := osgraphtest.BuildGraph("../../../api/graph/test/dueling-rcs.yaml")
+	if err != nil {
+		t.Fatalf("unexpected error: %v", err)
+	}
+
+	kubeedges.AddAllManagedByRCPodEdges(g)
+
+	if e, a := 2, len(FindDuelingReplicationControllers(g)); e != a {
```

Comment:
Same comment for all the Find* tests: verify the marker fields (especially the relations).